systemtest/jpsreport_tests/CMakeLists.txt

```diff
@@ -34,6 +34,15 @@ add_test(NAME J08_corridor_low_cut_off COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURR
 add_test(NAME J09_corridor_high_cut_off COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/method_J/J09_corridor_high_cut_off/J09_corridor_high_cut_off.py ${jpsreport_exe})
 add_test(NAME J10_corridor_cut_off_velocity COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/method_J/J10_corridor_cut_off_velocity/J10_corridor_cut_off_velocity.py ${jpsreport_exe})
 
+add_test(NAME reference_cut_off_disabled COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/reference_tests/cut_off_disabled/test_cut_off_disabled.py ${jpsreport_exe})
+add_test(NAME reference_cut_off_edges COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/reference_tests/cut_off_edges/test_cut_off_edges.py ${jpsreport_exe})
+add_test(NAME reference_cut_off_enabled COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/reference_tests/cut_off_enabled/test_cut_off_enabled.py ${jpsreport_exe})
+add_test(NAME reference_cut_off_large COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/reference_tests/cut_off_large/test_cut_off_large.py ${jpsreport_exe})
+add_test(NAME reference_cut_off_small COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/reference_tests/cut_off_small/test_cut_off_small.py ${jpsreport_exe})
+add_test(NAME reference_methods_ABC COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/reference_tests/methods_ABC/test_methods_ABC.py ${jpsreport_exe})
```

Comment:
Didn't we just write reference tests for Method D, I and J? Do we have some for ABC?

Reply:
The original inifiles executed Method A,B,C as well. I separated this from testing of D,I,J